### PR TITLE
Issue #15218: formatted input file for section 3.2 package statement to follow google style guide rules

### PR DESCRIPTION
**Pull request by Zopsss**
#15218

### CODE REVIEW

File: src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule32packagestatement/InputPackageStatement.java [hunk 1]
```diff
@@ -2,9 +2,13 @@
...
+  // -----------------------------------------------------------------------------------------------------
+  // violation above 'Line is longer than 100 characters (found 106).'
 
-  private int[] mInts = new int[] {1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24};
-  // violation above 'Line is longer than 100 characters (found 174).'
+  private int[] mInts =
+      new int[] {
+        1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 12,
+        13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24
+      };
```
Comment (Zopsss) [inline src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule32packagestatement/InputPackageStatement.java:13]: google formatter changed this code to be in column limit 100, this code was intentionally put here to show that column limit rule does not applies to package statements but it applies to other code.

Should I revert this and add back the violation?

Comment (romani) [reply src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule32packagestatement/InputPackageStatement.java:13]: Yes, format whole file and revert certain lines to keep violation.

Comment (romani) [reply src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule32packagestatement/InputPackageStatement.java:13]: >3.2 Package statement
The package statement is not line-wrapped. The column limit (Section 4.4, [Column limit: 100](https://checkstyle.org/styleguides/google-java-style-20180523/javaguide.html#s4.4-column-limit)) does not apply to package statements.

This violation is irrelevant for this chapter, we can loose it

**Review by romani — APPROVED** [review]
Ok to merge.

@rdiachenko , please reconsider fix.

**Review by rdiachenko — APPROVED** [review]
lgtm

**Pull request merged**